pkg/tsdb/loki/streaming.go

```diff
@@ -13,8 +13,6 @@ import (
 	"github.com/gorilla/websocket"
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 	"github.com/grafana/grafana-plugin-sdk-go/data"
-
-	"github.com/grafana/grafana/pkg/services/featuremgmt"
 )
 
 func (s *Service) SubscribeStream(_ context.Context, req *backend.SubscribeStreamRequest) (*backend.SubscribeStreamResponse, error) {
```

Comment:
I don't know this code base well enough to know if this method is used for the normal Tail operation done by Loki in explore?

If this method is used by the normal Tail operation from explore, this PR cannot be merged or it will break that functionality.

Reply:
hi @slim-bean , thanks for the comment, the answer is `no`. the normal Tail operation is running through a different part of the code.